tools/code_format/check_format.py

```diff
@@ -117,6 +117,8 @@
 EXCEPTION_DENYLIST = (
     "./source/common/http/http2/codec_impl.h", "./source/common/http/http2/codec_impl.cc")
 
+# Files that are allowed to use try without main thread assertion.
+# TODO(chaoqin-li1123): remove try catch pattern in dns_impl.cc.
 RAW_TRY_ALLOWLIST = (
     "./source/common/common/regex.cc", "./source/common/common/thread.h",
     "./source/common/network/utility.cc", "./source/common/network/dns_impl.cc")
```

Comment:
were you going to remove dns_impl.cc form this list, then, and change the `try` to `TRY_NEEDS_AUDIT`?

Reply:
I may not have the bandwidth to do this in the near future:)

Reply:
I'm not talking about following up on removing the try-catch pattern entirely. I'm just talking about leaving it in its current AUDIT state rather than the allow-list mechanism. This reversion shouldn't take more than a few minutes.

Reply:
Thanks, will do.